openupgrade_scripts/scripts/event/18.0.1.9/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,64 @@
+---Models in module 'event'---
+model event.question (moved from website_event)
+model event.question.answer (moved from website_event)
+model event.registration.answer (moved from website_event)
+---Fields in module 'event'---
+event        / event.event              / badge_format (selection)      : selection_keys is now '['96x134', '96x82', 'A4_french_fold', 'A6', 'four_per_sheet']' ('['A4_french_fold', 'A6', 'four_per_sheet']')
+event        / event.event              / general_question_ids (one2many): previously in module website_event
```

Comment:
group all things that come from website_event in one block.

Reply:
I disagree with changing work file order because this destroys seeing in one glance if there's something to do after generating a new changes file

Reply:
The diff is seen in the original file, not in the work file, so if new changes, you incorporate such changes to the work file when applicable, so no problem in reorder lines, and I prefer it to group them in logical blocks, not how the analysis module spit it.